client/profile-wizard/steps/store-details.js

```diff
@@ -167,67 +177,110 @@ class StoreDetails extends Component {
-											this.onContinue( values )
-										}
-										onClose={ () =>
-											this.setState( {
-												showUsageModal: false,
-											} )
-										}
-									/>
-								) }
+						<span className="woocommerce-profile-wizard__tooltip-icon">
+							<Icon icon="info-outline" size={ 16 } />
+						</span>
+					</Tooltip>
+				</H>
+
+				<Form
+					initialValues={ this.initialValues }
+					onSubmitCallback={ this.onSubmit }
+					validate={ validateStoreAddress }
+				>
+					{ ( {
+						getInputProps,
+						handleSubmit,
+						values,
+						isValidForm,
+						setValue,
+					} ) => (
+						<Card>
+							{ showUsageModal && (
+								<UsageModal
+									onContinue={ () =>
+										this.onContinue( values )
+									}
+									onClose={ () =>
+										this.setState( {
+											showUsageModal: false,
+										} )
+									}
+								/>
+							) }
+							<CardBody>
 								<StoreAddress
 									getInputProps={ getInputProps }
 									setValue={ setValue }
 								/>
+							</CardBody>
 
-								<div className="woocommerce-profile-wizard__client">
-									<CheckboxControl
-										label={ __(
-											"I'm setting up a store for a client",
-											'woocommerce-admin'
-										) }
-										{ ...getInputProps( 'isClient' ) }
-									/>
-								</div>
-
-								<Button
-									isPrimary
-									onClick={ handleSubmit }
-									disabled={ ! isValidForm }
-								>
-									{ __( 'Continue', 'woocommerce-admin' ) }
-								</Button>
-							</Fragment>
-						) }
-					</Form>
-				</Card>
+							<CardFooter>
+								<FlexItem align="center">
+									<div className="woocommerce-profile-wizard__client">
+										<CheckboxControl
+											label={ __(
+												"I'm setting up a store for a client",
+												'woocommerce-admin'
+											) }
+											{ ...getInputProps( 'isClient' ) }
+										/>
+									</div>
+								</FlexItem>
+							</CardFooter>
+
+							<CardFooter justify="center">
+								<FlexItem>
+									<div className="woocommerce-profile-wizard__submit">
+										<Button
+											isPrimary
+											onClick={ handleSubmit }
+											disabled={ ! isValidForm }
+										>
+											{ __(
+												'Continue',
+												'woocommerce-admin'
+											) }
+										</Button>
+									</div>
+								</FlexItem>
+							</CardFooter>
+						</Card>
+					) }
+				</Form>
+				<div className="woocommerce-profile-wizard__footer">
+					<a
+						className="woocommerce-profile-wizard__footer-link"
+						href={ adminUrl }
+					>
+						{ __( 'Skip setup wizard', 'woocommerce-admin' ) }
+
+						<Tooltip
+							text={ __(
+								'Manual setup is only recommended for\n experienced WooCommerce users or developers.',
+								'woocommerce-admin'
+							) }
+						>
+							<span className="woocommerce-profile-wizard__tooltip-icon">
+								<Icon icon="info-outline" size={ 16 } />
+							</span>
+						</Tooltip>
```

Comment:
I'm wondering if the tooltip should be outside of the link. Especially with the `cursor: pointer;`, it looks like you're supposed to click the info icon to get more info, but that makes you skip the profiler.

Reply:
Yep good point. Will fix.